jdaviz/core/template_mixin.py

```diff
@@ -1463,6 +1462,35 @@ def _sort_by_icon(items_dict):
 
         self._apply_default_selection()
 
+    def update_wcs_only_filter(self, wcs_only):
+        """
+        The layers that are populated in LayerSelect.choices
+        will be either WCS-only layers (for setting viewer orientation)
+        or non-WCS-only layers (for "real data"). This method toggles
+        the layer choices by adjusting the layer filters on this
+        LayerSelect instance.
+
+        Parameters
+        ----------
+        wcs_only : bool
+            `True` will filter only the WCS-only layers, `False` will
+            give the non-WCS-only layers.
+        """
+        def is_wcs_only(data):
+            return data.meta.get(self.app._wcs_only_label, False)
+
+        filter_names = [getattr(filt, '__name__', '') for filt in self.filters]
+
+        if not wcs_only and 'is_wcs_only' in filter_names:
+            self.filters.remove(*[filt for filt in self.filters
+                                  if getattr(filt, '__name__', '') == 'is_wcs_only'])
+        elif wcs_only and 'is_wcs_only' not in filter_names:
+            self.add_filter(is_wcs_only)
+
+    @property
+    def only_wcs_layers(self):
```

Comment:
How is this different from `get_wcs_only_layer_labels`? Please clarify in a docstring.